packages/theme/src/cli/commands/theme/check.ts

```diff
@@ -97,6 +123,59 @@ Excludes checks matching any category when specified more than once`,
 
   async run(): Promise<void> {
     const {flags} = await this.parse(Check)
+
+    if (flags['dev-preview']) {
+      if (flags.init) {
+        await initConfig(flags.path)
+
+        // --init should not trigger full theme check operation
+        return
+      }
+
+      if (flags.version) {
+        outputInfo(themeCheckPackage.version)
+
+        // --version should not trigger full theme check operation
+        return
+      }
+
+      if (flags.print) {
+        await outputActiveConfig(flags.config, flags.path)
+
+        // --print should not trigger full theme check operation
+        return
+      }
+
+      if (flags.list) {
+        await outputActiveChecks(flags.config, flags.path)
+
+        // --list should not trigger full theme check operation
+        return
+      }
```

Comment:
Shouldn't these be subcommands of theme check instead of flags?
Like:
```bash
shopify theme check list
shopify theme check version
...
```

Reply:
Also re: version, we should adopt the convention.
```
$ git --version
git version 2.42.0
hub version refs/heads/master
$ npm --version
10.1.0
$ yarn --version
1.22.19
$ gh --version
gh version 2.35.0 (2023-09-19)
https://github.com/cli/cli/releases/tag/v2.35.0
$ ruby --version
ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin22]
```